src/narrow-screen/components/MenuCategoryPanel.js

```diff
@@ -8,9 +8,24 @@ const DishList = styled.div`
   z-index: 1;
 `;
 
+const CategoryTitle = styled.div`
+  font-family: Source Serif Pro;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 33px;
+  letter-spacing: 0.02em;
+  color: #000000;
+  flex: none;
+  order: 0;
+  align-self: center;
+  margin: 10px 0px;
+`;
+
 export default function(props) {
   return (
     <DishList>
+      <CategoryTitle>{props.category}</CategoryTitle>
```

Comment:
I don't think this is working...?

Reply:
should be fixed, think the merge messed something up